.github/workflows/e2e-pr.yml

```diff
@@ -66,3 +66,17 @@ jobs:
               repo: "openmct",
               body: 'Failure ❌ ! Build artifacts are here: https://github.com/nasa/openmct/actions/runs/' + context.runId
```

Comment:
can you update this comment to use the octokit/request-action "while you're in there"

Reply:
I updated it all to use `github-script` instead. I think this is actually the canonical way to accomplish such tasks-- plus `octokit/request-action` was less readable and creating false warnings in our GHA logs.